.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL = 'mongodb://127.0.0.1:27017/smart-offer'
```

Comment:
You should never upload the `env` file in your Github project, add this your `.gitignore`

Reply:
Removed from the Repo, thanks